qiskit/pulse/model/__init__.py

```diff
@@ -0,0 +1,83 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+.. _pulse-logical-elements-frames:
+
+=======================================
+Logical Elements & Frames (:mod:`qiskit.pulse.logical_elements_frames`)
+=======================================
+
+Pulse is meant to be agnostic to the underlying hardware implementation, while still allowing
+low-level control. Qiskit Pulse's logical element and frames are meant to create a flexible framework
+to define where pulses instructions are applied, and what would be their carrier frequency and phase
+(because typically AC pulses are used). Each :class:`LogicalElement` represents a separate component
+in the quantum system on which instructions could be applied. On the other hand, each :class:`Frame`
+represents a frequency and phase duo for the carrier of the pulse.
+
+This logical and virtual representation allows the user to write template pulse
+programs without worrying about the exact details of the HW implementation (are the pulses to be played
```

Comment:
```suggestion
programs without worrying about the exact details of the hardware implementation (are the pulses to be played
```

(or introduce "HW" with the "hardware" use above).